### DISCUSSION TITLE
IPv6 support

### DISCUSSION THREAD

**Discussion by pschmiedel** [diasurgical/DevilutionX#1617] [Ideas]
With many ISPs using [DS-Lite](https://en.wikipedia.org/wiki/IPv6_transition_mechanism#Dual-Stack_Lite_.28DS-Lite.29), it would be nice if the game natively supported IPv6 for multiplayer. I'm currently tunneling IPv4 through an IPv6-VPN, which works, but is a hassle to setup for friends. Would be much nicer if they could use my IPv6 address / dynamic hostname.

**Comment by qndel** [comment]
I think we already support hostnames, no clue if it will work with IPv6 or not

**Reply by AJenbo** [reply]
We already support IPv6

**Reply by pschmiedel** [reply]
Ok, it doesn't seem to work for me. I keep getting "Connect: no connection could be made because the target machine actively refused it". I've tried:
* different IPv6 formats (including ::1)
* a dynamic hostname
* setting the "Bind Address" in the .ini file to an IPv6
* disabling the firewall
I'm on Windows btw.
